build.js

```diff
@@ -1,6 +1,16 @@
+// Copyright 2021 Record Replay Inc.
```

Comment:
This copyright should have the same BSD license notice that other chromium files and files we've written have.

Reply:
Woops! I was following the wrong project guidelines.
Fixed now.